code/game/objects/items/weapons/tools.dm

```diff
@@ -332,8 +332,8 @@
 		return
 	..()
 
-/obj/item/tool/weldingtool/proc/do_weld(var/mob/user, var/atom/thing, var/time, var/fuel_cost)
```

Comment:
This is a really hacky way to approach this fix, there has got to be a better way than adding an arg here only used for welding walls

Reply:
Fixed